pkg/model/iam.go

```diff
@@ -171,13 +171,11 @@ func (b *IAMModelBuilder) buildIAMTasks(igRole kops.InstanceGroupRole, iamName s
 		}
 
 		// Create External Policy tasks
-		{
+		if b.Cluster.Spec.ExternalPolicies != nil {
```

Comment:
```suggestion
		if !shared {
```

Reply:
This also works.

Reply:
Thanks for verifying @kesor. Can you update the PR with the suggested change and rebase to remove the "contains-merge-commits" label?

Reply:
Done! Any chance to see this in `v1.18.1`?

Reply:
Unfortunately a bit late for [v1.18.1](https://github.com/kubernetes/kops/releases/tag/v1.18.1), but should be part of v1.18.2.